src/main/webapp/style.css

```diff
@@ -148,3 +148,33 @@ ul {
   padding: 1rem 0;
   text-align: center;
 }
+
+.hiddenButton {
+  border: solid #333;
+  border-width: 0 5px 5px 0;
+  float: right;
+  padding: 5px;
+}
+
+.paginationButton {
+  border: solid white;
```

Comment:
I recommend we set the border width here. Also instead of using <code>white</code>, you could use the CSS variable <code>primary</code> setup in <code>.root</code>. This way the colour can be adjusted with the rest of the theme.

Reply:
Done, thank you!